### ISSUE TITLE
devtools does not show fonts using css "font:", but it does with css "font-family"

### ISSUE THREAD

**Issue by SuperNickkk** [nuxt/fonts#291]
I created a minimal nuxt setup. Installed nuxt/fonts module.  

Fonts referenced by css font-family appear in the fonts devtools panel.
Fonts referenced by css font do not appear.

This is not a new issue.  I have noticed this behaviour before.  I have just recently had time to created a minimal example and work out what (it seems to me) is not working.

**npm list:**
@nuxt/fonts@0.9.1
nuxt@3.13.2
vue-router@4.4.5
vue@3.5.8

**app.vue:**
```
<template>
  <div>
    <NuxtRouteAnnouncer />
    <p>This is a test</p>
  </div>
</template>
<style>
p {
      font: 1.25rem/1.75rem Lato, serif;
/*       font-family: Lato, serif; */
}
</style>
```

It seems to me that toggling betwen "font:" and "font-family:" in the css makes Lato (or any other font) appear or disappear in the devtools fonts tab.  

Of course, it's also possible that I might be doing something very stupid.  Or that the fonts module does not respect "fonts:" by design.

Sometimes "npm run dev" is required to show this behaviour.

Examples:

![image](https://github.com/user-attachments/assets/e0f1267f-f3f3-47c1-b3d4-f45a92951f6b)

![image](https://github.com/user-attachments/assets/550f947f-bb52-44dd-a4af-e15f8da99c43)

**Comment by SuperNickkk** [comment]
Just to add, the issue is still there in @nuxt/fonts 0.9.2

With thanks for your help. . .

**Comment by danielroe** [comment]
You're right, `nuxt/fonts` does not scan the `font:` declaration. It should.
